jOOQ/pom.xml

```diff
@@ -115,24 +115,11 @@
             <scope>provided</scope>
             <optional>true</optional>
         </dependency>
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
```

Comment:
Please don't remove this whitespace, as documented in the contribution guidelines: https://github.com/jOOQ/jOOQ/blob/main/CONTRIBUTING.md